storage/Devices/Bcache.h

```diff
@@ -33,6 +33,14 @@ namespace storage
     class BcacheCset;
 
 
+    /**
+     * The Cache mode attribute.
+     */
+    enum class CacheMode {
```

Comment:
IMHO this is specific to bcache so it should be moved inside the Bcache namespace/class.

Reply:
It is consistent with rest of library which also has it outside like MD raid level and others.

Reply:
Uggggggh OK.